domain/src/test/java/org/fao/geonet/domain/ISODateTest.java

```diff
@@ -448,4 +453,34 @@ public void testSub() throws Exception {
         org.junit.Assert.assertEquals(-60, isoDate2.timeDifferenceInSeconds(isoDate1));
     }
 
+
+    @Test
+    public void testZ() throws Exception {
+        ISODate isoDate = new ISODate("2019-06-01T00:00Z");
+        Instant instant = isoDate.toDate().toInstant().truncatedTo(ChronoUnit.SECONDS);
+
+        ZonedDateTime expectedDateTime =
+            ZonedDateTime.of(
+                2019, 6, 1, 0, 0, 0, 0, ZoneId.of("Z").normalized());
+        Instant expected = expectedDateTime.toInstant().truncatedTo(ChronoUnit.SECONDS);
+        assertEquals( "Z", expected, instant );
+
+        ISODate pstDate = new ISODate("2019-06-01T00:00-07:00");
+        instant = pstDate.toDate().toInstant().truncatedTo(ChronoUnit.SECONDS);
+
+        ZonedDateTime pstDateTime =
+            ZonedDateTime.of(
+                2019, 6, 1, 0, 0, 0, 0, ZoneId.of("America/Vancouver").normalized());
+        expected = pstDateTime.toInstant().truncatedTo(ChronoUnit.SECONDS);
+        assertEquals( "PST", expected, instant );
```

Comment:
It is incorrect to use PST since it is PDT roughly half of the year.
Generally prefer to use the zone name i.e. in "Pacific Time Zone" or "Pacific"
Same for EST below.

Reply:
Thanks, reading the ZoneId docs I am going to use `UTC-04:00` as I am more interested in testing differences in timezone than anything else.